contracts/registry/JurorsRegistry.sol

```diff
@@ -130,17 +129,14 @@ contract JurorsRegistry is ControlledRecoverable, IJurorsRegistry, ERC900, Appro
     /**
     * @dev Constructor function
     * @param _controller Address of the controller
-    * @param _jurorToken Address of the ERC20 token to be used as juror token for the registry
     * @param _totalActiveBalanceLimit Maximum amount of total active balance that can be held in the registry
     */
-    constructor(Controller _controller, ERC20 _jurorToken, uint256 _totalActiveBalanceLimit)
+    constructor(Controller _controller, uint256 _totalActiveBalanceLimit)
         ControlledRecoverable(_controller)
         public
     {
         // No need to explicitly call `Controlled` constructor since `ControlledRecoverable` is already doing it
-        require(isContract(address(_jurorToken)), ERROR_NOT_CONTRACT);
-
-        jurorsToken = _jurorToken;
+        jurorsToken = _getConfigAt(0).fees.token;
```

Comment:
We should probably document in the docs that this deployment will only work with the same token for fees token and juror token.